src/components/Card.js

```diff
@@ -48,12 +38,36 @@ export default class Card {
     });
   }
 
-  _handleDeleteButton() {
+  cardIsLiked() {
+    return this._isLiked;
+  }
+
+  handleDeleteButton = () => {
     this._cardElement.remove();
     this._cardElement = null;
-  }
+  };
 
-  _handleLikeButton() {
+  handleLikeButton = () => {
```

Comment:
Needs correction: you don't need to toggle the likes button anymore, you should update it's state according to the server response.

The code for rendering likes should look like this:

```js
// in Card class 

// call this method from index.js after successful api response in `then` block
setIsLiked(isLiked) { 
   // set instance variable
    this._isLiked = isLiked; 
    this._renderLikes(); 
} 

isLiked() {
  return this._isLiked;
}

_renderLikes() {
  if (this._isLiked) {
    // add active class to the like button
  } else {
    // remove active class from the like button
  }
}

createCard() { 
  ...
  // render initial likes, you saved them in the constructor
  this._renderLikes()
}

// index.js
if (card.isLiked()) {
  api.dislikeCard(...).then(response => card.setIsLiked(response._isLiked)).catch(() => /*handle error*/)
} else {
 // the same for adding like
}
```